cg/meta/upload/mip/mip_rna.py

```diff
@@ -0,0 +1,43 @@
+"""MIP-RNA upload API"""
+
+import logging
+
+import click
+
+from cg.cli.upload.scout import upload_rna_to_scout
+from cg.cli.upload.clinical_delivery import clinical_delivery
+from cg.constants import DataDelivery
+from cg.meta.workflow.mip_rna import MipRNAAnalysisAPI
+from cg.meta.upload.upload_api import UploadAPI
+from cg.models.cg_config import CGConfig
+from cg.store import models
+
+LOG = logging.getLogger(__name__)
+
+
+class MipRNAUploadAPI(UploadAPI):
+    """MIP-RNA upload API"""
+
+    def __init__(self, config: CGConfig):
+        self.analysis_api: MipRNAAnalysisAPI = MipRNAAnalysisAPI(config)
+        super().__init__(config=config, analysis_api=self.analysis_api)
+
+    def upload(self, ctx: click.Context, case_obj: models.Family, restart: bool) -> None:
+        """Uploads MIP-RNA analysis data and files"""
+
+        analysis_obj: models.Analysis = case_obj.analyses[0]
+        self.update_upload_started_at(analysis_obj)
```

Comment:
Use named args even if only 1. It is easier to understand what the function needs to run and more extendible in the  future